tools/ci/emscripten.meson.cross

```diff
@@ -0,0 +1,17 @@
+[binaries]
+c = 'emcc'
+cpp = 'em++'
+ar = 'emar'
+fortran = '/usr/bin/gfortran'
+
+cmake = ['emmake', 'cmake']
+sdl2-config = ['emconfigure', 'sdl2-config']
+
+[host_machine]
+system = 'emscripten'
+cpu_family = 'wasm32'
+cpu = 'wasm'
+endian = 'little'
+
+[properties]
+longdouble_format = 'IEEE_QUAD_LE'
```

Comment:
This is the only addition compared to the upstream `emscripten.meson.cross` file in Pyodide - necessary for now.

Reply:
Most of this file can be removed, only this is needed:
```ini
[properties]
longdouble_format = 'IEEE_QUAD_LE'
```